examples/react-native/src/Type.js

```diff
@@ -10,13 +10,13 @@ import {
   TouchableHighlight,
   Keyboard,
 } from 'react-native';
-import { InstantSearch } from 'react-instantsearch/native';
+import { InstantSearch } from 'react-instantsearch-native';
 import {
   connectRefinementList,
   connectSearchBox,
   connectRange,
   connectMenu,
-} from 'react-instantsearch/connectors';
+} from 'react-instantsearch-core';
```

Comment:
We should import from `react-instantsearch-native`. The `core` package is only used internally.